### ISSUE TITLE
YAML parsing error on config/database.yml while running web service

### ISSUE THREAD

**Issue by Coukaratcha** [tootsuite/mastodon#15226]
I got this error when I am trying to run web service using:

`/opt/mastodon/vendor/bundle/ruby/2.6.0/gems/railties-5.2.4.3/lib/rails/application/configuration.rb:190:in `rescue in database_configuration': YAML syntax error occurred while parsing /opt/mastodon/config/database.yml. Please note that YAML must be consistently indented using spaces. Tabs are not allowed. Error: (<unknown>): found character that cannot start any token while scanning for the next token at line 12 column 13 (RuntimeError)`

Other services are up.

I use podman and podman-compose on Fedora 31.

**Comment by Gargron** [comment]
You seem to have a tab instead of spaces in config/database.yml

**Comment by Coukaratcha** [comment]
Yes, it seems so. But I did not edit this file, I have just pull the last version from this repo. When I read the file, there are spaces for indentation (2 spaces to be exact). So I do not understand why the parsing failed with this message.

**Comment by Gargron** [comment]
Maybe there is a tab in one of the environment variables that get rendered in that file?

**Comment by Coukaratcha** [comment]
I checked line 12 of database.yml and it called DB_PASSWORD. The password I chose was very long, so I tried to change it with a shorter one and it worked. Thanks for leading me to this conclusion.

Now I have another issue. When I run db:migrate job I have the following error message:
`ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?`

Despite I have set the variable REDIS_PASSWORD in .env.production file. Do I miss something?

And for your information, now web service is running and I can get the url of my instance (I got the same error in the logs of the service, but it is a progress).
